flu/flu.prepare.py

```diff
@@ -53,9 +61,10 @@ def make_config(lineage, resolution, params):
 
     return {
         "dir": "flu",
-        "file_prefix": "flu_{}".format(lineage),
+        "file_prefix": "flu_%s"%lineage,
+        "dataset":params.dataset,
         "segments": params.segments,
-        "resolution": resolution,
+        "resolution": time_id,
```

Comment:
This felt like a strange bit of naming. I like the inclusion of an explicit `time_interval`, but I would prefer to stick with `resolution` for the temporal identifier of the dataset. (I know the end result is the same)

Reply:
I don't feel strongly. I felt that `resolution` as a name is weird when using an arbitrary time interval.

Reply:
Yes. Agreed that `resolution` is a little weird in this case, but I think consistency of using `resolution` for time disambiguation outweighs the weirdness.

Reply:
why not collapse `resolution` and `dataset` into the `identifier`. nobody needs to know about resolution after we spec the time interval. same is true for lineage.